app/javascript/components/student/editor/ExercismMonacoEditor.tsx

```diff
@@ -44,18 +44,17 @@ export function ExercismMonacoEditor({
   value: string | null | undefined
   theme: string
 }) {
-  const handleEditorDidMount = (
-    editor: monacoEditor.editor.IStandaloneCodeEditor
-  ) => {
-    editor.addAction({
-      id: 'runTests',
-      label: 'Run tests',
-      keybindings: [monacoEditor.KeyCode.F2],
-      run: onRunTests,
-    })
+  const startLanguageClient = useCallback(() => {
+    const languageServerUrl = document.querySelector<HTMLMetaElement>(
+      'meta[name=language-server-url]'
+    )?.content
```

Comment:
It always _must_ be present. You could actually throw when it's not! (I would). I know you log an error a little bit later, but I would throw.

Reply:
Thanks! I am throwing the error now.